### DISCUSSION TITLE
Where is the Vault Guardians official report ?

### DISCUSSION THREAD

**Discussion by viniciuscoassis** [Cyfrin/security-and-auditing-full-course-s23#244] [General]
I finally finished the vault guardians security review, but can't find the official report with all the findings, this is so I can get my feedback of what I got right/wrong/missed.

There is one report on audit-data but it's supposed to be the issues the protocol was aware of beforehand right ? how about the oficial one ?

**Comment by EngrPips** [comment]
How do you mean the official one?
